src/hooks/usePrevious/__stories__/usePrevious.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import usePrevious from "../index";
+import UsePrevious from "./hooksDummyComponents/UsePrevious";
+import Button from "components/Button/Button";
+import { Counter } from "components";
+import Flex from "components/Flex/Flex";
+import "./usePrevious.stories.scss";
+
+<Meta title="Hooks/usePrevious" component={UsePrevious} />
+
+# usePrevious
+
+- [Overview](#overview)
+- [Arguments](#arguments)
+- [Feedback](#feedback)
+
+## Overview
+
+Hook for keeping previous state value.
+
+<Canvas>
+  <Story name="Overview">
+    {() => {
+      const [count, setCount] = useState(1);
+      const prevCount = usePrevious(count);
+      const incrementButtonOnClick = () => {
+        setCount(prevValue => prevValue + 1);
+      };
+      return (
+        <div>
```

Comment:
We can remove the outside div because it does not have any style / classes inside and therefore not needed

Reply:
I've used div parent as with this` return ()` syntax we have to return single parent element always. We could use `return [<>, <>]` syntax though, but I believe it's a bad option in most cases
Replaced div with Flex for more clarity and flexibility. Is that ok?

Reply:
I saw you removed it and the flex become the single parent element (which it was there before so its ok)
if you don't need flex and just looking for wrapping your return react code, you can return react fragment like this: `<><div>1</div><div>2</div></>`